libsolidity/formal/SMTChecker.cpp

```diff
@@ -29,8 +29,8 @@
 #include <boost/optional.hpp>
 
 using namespace std;
-using namespace dev;
 using namespace langutil;
+using namespace dev;
```

Comment:
Shouldn't these be sorted in the reverse order? From Solidity-specific to more general?

Reply:
I thought it was general  \/ specific? Anyway updated putting `dev` back above `langutil`